roles/riemann-docker/defaults/main.yml

```diff
@@ -5,7 +5,7 @@ riemann_tcp_port: "5555"
 riemann_udp_port: "5555"
 
 riemann_influxdb_version: "0.9"
-riemann_influxdb_host: "127.0.0.1"
+riemann_influxdb_host: "{{ ansible_ssh_host }}"
```

Comment:
Why is {{ ansible_ssh_host }} better than 127.0.0.1? This has to be defined elsewhere anyhow. Stating the property here is purely for documentation purpose. In that sense, 127.0.0.1 (an IP address) is more explanatory than {{ ansible_ssh_host }}, in my opinion.